Makefile

```diff
@@ -26,9 +27,17 @@ clean:
 	$(GOCMD) clean
 	rm -rf $(BUILD_DIR)
 
+# TODO: For now we support only the linux version of tailwindcss, has to be updated in the future to support Windows and MacOS as well.
 install-tools:
 	go install github.com/pressly/goose/v3/cmd/goose@v3.18.0
 	go install github.com/a-h/templ/cmd/templ@v0.2.543 
 	go install github.com/cosmtrek/air@v1.49.0 
 
-.PHONY: all build test clean
+install-tailwindcss:
+	if [ ! -f tailwindcss ]; then \
+		wget -q https://github.com/tailwindlabs/tailwindcss/releases/latest/download/tailwindcss-linux-x64 \
```

Comment:
We should probably be specific about the tailwind version here and do a checksum